force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -39,18 +40,15 @@ public with sharing class ClassesObjectsMethods {
      * @return A list containing the length of the string, a substring of the string, and the string in lower case.
      */
     public static List<String> practiceStringMethods() {
-        // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        List<String> result = new List<String>();
+        String s1 = 'Hello, Salesforce!';
+        Integer strLength = s1.length();
+        String sub1 = s1.subString(0, 5);
+        String s1LowerCase = s1.toLowerCase();
+        result.add(String.valueOf(strLength));
+        result.add(sub1);
+        result.add(s1LowerCase);
+        return result;
```

Comment:
Good work here.  The only thing I would do differently is add the results of the String method calls to the list directly. If I don't need the data in a variable for anything else and it does not help with readability, I try not to use them. 
```
   public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
 ```